kernel/packages/entryPoints/unity.ts

```diff
@@ -117,7 +121,12 @@ initializeUnity(container)
     }
 
     if (!NO_MOTD) {
-      i.ConfigureHUDElement(HUDElementID.MESSAGE_OF_THE_DAY, { active: false, visible: true }, worldConfig.messageOfTheDay)
+      const messageOfTheDay = await waitForMessageOfTheDay()
```

Comment:
Do we need to await here blocking the initialization below? Couldn't this execute in parallel, i.e with an anonymous async function?